source/simulator/core.cc

```diff
@@ -2153,6 +2167,13 @@ namespace aspect
 
               pcout << std::endl;
 
+              // output relative residuals
```

Comment:
You are right, I don't really see a good reason to keep the absolute residuals for `iterated_IMPES`. If everyone is okay with that, I would just throw them out.